src/wp-includes/load.php

```diff
@@ -1807,6 +1807,10 @@ function wp_start_scraping_edited_file_errors() {
 	$key   = substr( sanitize_key( wp_unslash( $_REQUEST['wp_scrape_key'] ) ), 0, 32 );
 	$nonce = wp_unslash( $_REQUEST['wp_scrape_nonce'] );
 
+	if ( get_transient( 'scrape_key_' . $key ) === false ) {
```

Comment:
```suggestion
	if ( false === get_transient( 'scrape_key_' . $key ) ) {
```

Add Yoda Condition check.